openpype/hosts/nuke/plugins/publish/extract_review_data_mov.py

```diff
@@ -8,7 +8,7 @@
 from openpype.hosts.nuke.api.lib import maintained_selection
 
 
-class ExtractReviewDataMov(publish.Extractor):
```

Comment:
Label and file name should be also changed to `Extract Review Baking Streams` and `extract_review_baking_streams.py`

Reply:
Also we will need to figure out the way to do the backward compatibility for the old plugin settings inside of the plugin. You could use classmethod `self.apply_settings(project_settings)` to use similar approach as it is used here https://github.com/ynput/OpenPype/pull/5623/files#diff-79438371778038156c3abd11150aff9b31a510034f4891c40e16e6f55482885bR752-R762

Reply:
here is also some more information about apply_settings during plugin discovery in publishing `openpype\pipeline\publish\lib.py:500`